config/locales/en.yml

```diff
@@ -32,6 +32,8 @@
 en:
   help: 'Help'
   utils:
+    'yes': Yes
+    'no': No
```

Comment:
Oui, [bonjour](https://makandracards.com/makandra/24809-yaml-keys-like-yes-or-no-evaluate-to-true-and-false) 👋 

(Le YAML c'est infect.)